src/test/java/org/cyclonedx/parsers/XmlParserTest.java

```diff
@@ -102,7 +102,6 @@ public void testParsedObjects11Bom() throws Exception {
         final byte[] bomBytes = IOUtils.toByteArray(this.getClass().getResourceAsStream("/bom-1.1.xml"));
         final XmlParser parser = new XmlParser();
         final Bom bom = parser.parse(bomBytes);
-        Assert.assertEquals("1.1", bom.getSpecVersion());
```

Comment:
Same here and the others below, that assert was removed

Reply:
2e23776